docs/rsdl/rapid-pro-rsdl-abnf.md

```diff
@@ -37,51 +37,71 @@ modelElement = structuredType / enumType / service
 ### Structured Type
 
 ```ABNF
-structuredType       = %s"type" identifier "{" *typeMember "}"
+structuredType       = annotations %s"type" identifier "{" *structuredTypeMember "}"
 
-structuredTypeMember = property / operation ; property, bound action, or bound function
+structuredTypeMember = property / operation ; property, action, or function
 
-property             = *propertyAnnotation identifier ":" typeReference
+property             = annotations [propertyModifier] identifier ":" typeReference
 
-propertyAnnotation   = %s"@key"
+propertyModifier     = %s"key"
 
 typeReference        = typeName [ "?" ] / "[" typeName [ "?" ] "]"
 
 typeName             = builtInType / %s"Edm" "." identifier / qualifiedName
 
 builtInType          = %s"Boolean" / %s"Date" / %s"Datetime" / %s"Double" / %s"Integer" / %s"String"
 
-operation            = [ actionAnnotation ] identifier
+operation            = annotations [operationModifier] identifier
                        "(" [ parameter *("," parameter) ] ")"
                        [ ":" typeReference ]
 
-actionAnnotation     = %s"@action"
+operationModifier    = %s"action" / %s"function"
 
 parameter            = identifier ":" typeReference
```

Comment:
```suggestion
parameter            = annotations identifier ":" typeReference
```

Reply:
would like to address this separately in issue #176 . Please resolve if you agree.